chapter_generative-adversarial-networks/dcgan_vn.md

```diff
@@ -593,7 +599,10 @@ train(net_D, net_G, data_iter, num_epochs, lr, latent_dim)
 It aims to fix the “dying ReLU” problem and helps the gradients flow easier through the architecture.
 -->
 
-*dịch đoạn phía trên*
+* Kiến trúc DCGAN gồm có bốn tầng tích chập cho Bộ phân biệt và bốn tầng tích chập "sải bước một phần (*fractionally-strided*)" cho Bộ sinh.
```

Comment:
À fyi từ `fractionally-strided` trước mình thấy có ai đó dịch là `sải bước phân số` thì phải. 
Bản thân mình cũng không hiểu rõ nguồn gốc của cái tên này lắm nên cũng k biết dịch thế nào đúng hơn.

Reply:
Qua phần code thì có vẻ như 4 tầng tích chập có số stride khác nhau nên được gọi như vậy. Chỗ này em cũng không rõ lắm.

Reply:
nó là tên gọi khác cho `transposed convolution` thôi, chắc ngụ ý việc tầng này phóng to feature map với "stride < 1"

Reply:
Em cũng mới đọc qua, đúng là tên gọi này xuất phát từ việc coi `transposed convolution` tương đương với cnn có s < 1.
`sải bước phân đoạn` thì sao nhỉ?

Reply:
nếu vậy thì sẽ là `segmental stride`, không phù hợp.  Mình thấy `một phần` mang đủ nghĩa

Reply:
để mình bê qua discussion #2847